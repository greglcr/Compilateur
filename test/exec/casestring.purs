module Main where

import Prelude
import Effect
import Effect.Console

f :: String -> Effect Unit
f x = case x of "foo" -> log "cool"
                "bar" -> log "not cool"
                _ -> log "hmm"

main :: Effect Unit
main = do f "foo"
          f "bar"
          f "fo"
          f "fooo"
          f "fo@"
          f "gfdgdf"
