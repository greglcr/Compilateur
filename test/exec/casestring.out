cool
not cool
hmm
hmm
hmm
hmm
